cibyl/utils/filtering.py

```diff
@@ -127,21 +112,18 @@ def satisfy_range_match(model: Dict[str, str], user_input: Argument,
 
 
 def filter_topology(model: Dict[str, str], operator: str, value: str,
-                    component: str):
+                    component: str) -> bool:
     """Check whether model should be included according to the user input. The
     model should be added if the its topology is consistent with the components
     requested by the user (number of controller or compute nodes).
 
     :param model: model information obtained from jenkins
-    :type model: str
     :param operator: operator to filter the topology with
-    :type operator: str
     :param value: Value to use in the comparison
     :param value: str
```

Comment:
I just realised this lines should be removed, but were missing the type keyword before.

Reply:
Changed. Thanks!